CMakeLists.txt

```diff
@@ -52,7 +53,9 @@ ENDIF()
 
 IF(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 	SET( USE_ICONV ON )
-	FIND_PACKAGE(Security)
+	IF(USE_TLS STREQUAL "OpenSSL" OR USE_TLS STREQUAL "ON")
+		FIND_PACKAGE(Security)
```

Comment:
Shouldn't this be `IF(USE_TLS STREQUAL "SecureTransport" OR USE_TLS STREQUAL "ON")`?